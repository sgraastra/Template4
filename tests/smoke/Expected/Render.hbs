<script type="text/x-handlebars-template"> <ul> {{#ifE showFirst 'true'}} <li data-id="1"> <button>{{item}}</button> </li> {{/ifE}}{{#each items}} <li data-id="{{id}}"> <button class="{{#ifE selected 'true'}}active{{/ifE}}"> {{item}} </button> </li> {{/each}}{{#ifE showLast 'true'}} <li data-id="{{lastId}}"> <button>{{item}}</button> </li> {{/ifE}} </ul> </script> 