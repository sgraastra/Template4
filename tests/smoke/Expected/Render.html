<!DOCTYPE html> <html lang="en"> <head> <meta charset="UTF-8"> <meta name="viewport" content="width=device-width, initial-scale=1.0"> <title>Template4 Test</title> </head> <body> <h1>Testing Testing</h1> <h2>Replace &amp; Replace Raw</h2> <p>Hello &lt;strong style=&quot;color: #336699&quot;&gt;World!&lt;/strong&gt;.</p> <p>Hello <strong style="color: #336699">World!</strong>.</p> <h1>Condition Sets</h1>  <p>In 1, 2 or 3!</p>  <p>Quoted if-statement</p>  <p>In "Hello World!" or "Foo"</p>  <p>Either foo or bar: <strong>foo</strong></p>  <p> The following line should end with <strong>2ef7bde608ce5404e97d5f042f95f89f1c232871</strong>. <br/> Here are some literal [, ], { and } characters; [Hello] {World}! </p> <h1>Recursive Repeater</h1>  <ol>  <li> Level 1-1, value: correct foo! <ol>  <li> Level 2-1, value: correct foo! <ol>  <li> Level 3-1, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 3-2, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 2-2, value: correct foo! <ol>  <li> Level 3-1, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 3-2, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 1-2, value: correct foo! <ol>  <li> Level 2-1, value: correct foo! <ol>  <li> Level 3-1, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 3-2, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 2-2, value: correct foo! <ol>  <li> Level 3-1, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  <li> Level 3-2, value: correct foo! <ol>  <li> Level 4-1, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  <li> Level 4-2, value: correct foo! <ol>  <li> Level 5-1, value: correct foo! <ol>  </ol> </li>  <li> Level 5-2, value: correct foo! <ol>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  </ol> </li>  </ol>  <h1>Nested Sections &amp; Includes</h1> <p>Below, "Static include" and "Include template" and "Include template with spaces" should be present.</p> <p>
  Static include: <strong>Hello World!</strong>
</p>
<p> Include template: Foo &ndash; Hello World! &ndash; bar... </br> Hello World! </p>  <p > Lorem ipsum inside a section... </p> <p> Include template with <strong><em>spaces!!!</em></strong> Hello World!. </br> Hello World! </p>  <p style="font-weight:bold;"> Double-nested: Lorem ipsum in global scope (should be bold)... </p>  <h1>Repeater &amp; Scope</h1> <ul>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; One</li>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; Two</li>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; <strong>Global</strong></li>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; Three</li>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; <strong>Global</strong></li>  <li>2ef7bde608ce5404e97d5f042f95f89f1c232871 &ndash; <strong>Global</strong></li>  </ul> </body> </html> 